Header header
int size
float32[] object_distances